Realm/RLMArray.h

```diff
@@ -300,8 +300,3 @@
 
 @end
 
-
-@interface RLMArray (Private)
-- (instancetype)initWithObjectClassName:(NSString *)objectClassName;
```

Comment:
Doesn't this still need to be in the public header for swift stuff using the obj-c API?